### PR TITLE
Make chat participant pane optional in chat composite

### PR DESCRIPTION
**Pull request by JamesBurnside**
# What
* Disable participants in the chat pane by default
* Add optional property to show participants
* Some small responsive fixes to make the experience better on mobile

# Why
Participant pane is not responsive, also disabling by default makes the chat experience very streamlined

# How tested
Tested each composite experience in the storybook in large desktop view and small mobile view

### CODE REVIEW

**Review by alkwa-msft — APPROVED** [review]

File: packages/communication-react/review/communication-react.api.md [hunk 1]
```diff
@@ -590,6 +590,7 @@ export type ChatObjectMethodNames<TName extends string, T> = {
 
 // @public (undocumented)
```
Comment (prprabhu-ms) [inline packages/communication-react/review/communication-react.api.md:591]: Slightly tangential to this PR: Document this?

File: packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts [hunk 2]
```diff
@@ -54,8 +48,8 @@ export const chatHeaderContainerStyle = mergeStyles({
 });
 
 export const topicNameLabelStyle = mergeStyles({
-  fontSize: '1.1rem',
   lineHeight: '2.5rem',
+  fontSize: '1.1rem',
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts:52]: Why reorder? If you are reordering, consider alphabetizing.

File: packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx [hunk 3]
```diff
@@ -28,9 +28,20 @@ export type ChatCompositeProps = {
...
 
 export type ChatOptions = {
-  sendBoxMaxLength?: number; // Limit max send box length, when change viewport size
-  // messagesPerPage?: number; // Number of messages per page - smaller for better perf
-  // supportNewline: boolean; // Whether to support new line (shift+enter) in textArea, disable until ACS backend supports line switch
+  /** Choose to show the participant pane */
+  showParticipantPane?: boolean;
+  /** Set a max width of the send box */ // TODO: we should remove this.
+  sendBoxMaxLength?: number;
+  /**
```
Comment (prprabhu-ms) [inline packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx:35]: nitty: Just delete these commented out options?

There's always history. Dead code is a drag.

**Review by prprabhu-ms — APPROVED** [review]

File: packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts (hunk 2 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts:52]: unintentional change, was removed then put it back in - but slightly in the wrong place it seems

File: packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx (hunk 3 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/ChatComposite/ChatComposite.tsx:35]: Agree, removed and filed bugs in their place
https://skype.visualstudio.com/SPOOL/_workitems/edit/2508474
https://skype.visualstudio.com/SPOOL/_workitems/edit/2508476

File: packages/communication-react/review/communication-react.api.md (hunk 1 above)
Comment (JamesBurnside) [reply packages/communication-react/review/communication-react.api.md]: Added quick documentation

File: packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts (hunk 2 above)
Comment (JamesBurnside) [reply packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts]: removed change

**Pull request merged**